asciidoctor-gradle-jvm/src/gradleTest/complex-jvm-setup/build.gradle.kts

```diff
@@ -0,0 +1,61 @@
+import org.asciidoctor.gradle.jvm.AsciidoctorPdfTask
+import org.asciidoctor.gradle.jvm.AsciidoctorTask
+import org.asciidoctor.gradle.jvm.OutputOptions
+import org.asciidoctor.gradle.jvm.ProcessMode
+
+plugins {
+    id("org.asciidoctor.jvm.convert")  version "2.0.0-alpha.2"
+    id("org.asciidoctor.jvm.pdf") version "2.0.0-alpha.2"
+    id("com.gradle.build-scan") version "1.16"
+}
+
+repositories {
+    jcenter()
+}
+
+asciidoctorj {
+    setDiagramVersion("1.5.4.1")
```

Comment:
`diagramVersion = "1.5.4.1"` should be possible but isn't. 

That's because Kotlin is picky to when it allows a property. I think the problem could be solved like this

```diff
String getDiagramVersion() { ... }
- void setDiagramVersion(Object v) 
+ void setDiagramVersion(String v) 
```